model.py

```diff
@@ -182,7 +182,7 @@ def _inverted_res_block(inputs, expansion, stride, alpha, filters, block_id, ski
     if block_id:
         # Expand
 
-        x = Conv2D(expansion * in_channels, kernel_size=1, padding='same',
+        x = Conv2D(int(expansion * in_channels), kernel_size=1, padding='same',
```

Comment:
I believe is better to change line 177 to `in_channels = inputs.shape[-1].value` instead of the actual change, but at the end is the same.

Reply:
What environment are you running this in? When I run it in Python 3.7 and Tensorflow 1.14, it is saying `AttributeError: 'int' object has no attribute 'value'`

Reply:
Python 3.6